src/main/resources/resource/WebGui/app/service/js/OpenCVGui.js

```diff
@@ -38,133 +38,14 @@ angular.module('mrlapp.service.OpenCVGui', []).controller('OpenCVGuiCtrl', ['$sc
      */
     $scope.filterMetaData = {
         'AdaptiveThreshold': {
-            algorithm: 'mean',
-            blockSize: {
-                options: {
-                    floor: 3,
-                    ceil: 13,
-                    minLimit: 3,
-                    maxLimit: 13,
-                    value: 11,
-                    step: 2,
-                    showTicks: true,
-                    onChange: function(id) {
-                        $scope.setFilterState()
-                    }
-                }
-            },
-            param1: {
-                options: {
-                    floor: -10,
-                    ceil: 10,
-                    minLimit: -10,
-                    maxLimit: 10,
-                    showTicks: true,
-                    value: 2,
-                    step: 1,
-                    onChange: function(id) {
-                        $scope.setFilterState()
-                    }
-                }
-            },
-
+            algorithm: 'mean'
```

Comment:
what happened to these settings?   did this stuff move somewhere else?

Reply:
this is replacement of rzslider custom library with standard html5 input/range ..
the defaults rightfully come from the java filters themselves